app/jobs/itms/fetch_all_data.rb

```diff
@@ -0,0 +1,69 @@
+class Itms::FetchAllData < ApplicationJob
+  queue_as :itms
+
+  def self.perform_all_harvesters
+    harvesters = [
+      {
+        harvester: Itms::AccountingDocumentsHarvester,
+        object: Itms::AccountingDocument
+      },
+      {
+        harvester: Itms::ApprovedZonfpHarvester,
+        object: Itms::ApprovedZonfp
+      },
+      {
+        harvester: Itms::DeclaredAppealsHarvester,
+        object: Itms::DeclaredAppeal
+      },
+      {
+        harvester: Itms::FinishedProjectsHarvester,
+        object: Itms::FinishedProject
+      },
+      {
+        harvester: Itms::OngoingProjectsHarvester,
+        object: Itms::OngoingProject
+      },
+      {
+        harvester: Itms::OperationalProgramsHarvester,
+        object: Itms::OperationalProgram
+      },
+      {
+        harvester: Itms::PaidZopHarvester,
+        object: Itms::PaidZop
+      },
+      {
+        harvester: Itms::PlannedAppealsHarvester,
+        object: Itms::PlannedAppeal
+      },
+      {
+        harvester: Itms::ProcurementsHarvester,
+        object: Itms::Procurement
+      },
+      {
+        harvester: Itms::ReceivedZonfpHarvester,
+        object: Itms::ReceivedZonfp
+      },
+      {
+        harvester: Itms::RejectedZonfpHarvester,
+        object: Itms::RejectedZonfp
+      },
+      {
+        harvester: Itms::RejectedZopHarvester,
+        object: Itms::RejectedZop
+      },
+      {
+        harvester: Itms::SubmittedZopHarvester,
+        object: Itms::SubmittedZop
+      },
+    ]
+
+    harvesters.each do |harvester|
+      Itms::FetchAllData.perform_later(harvester[:harvester].to_s, harvester[:object].to_s)
+    end
+  end
+
+  def perform(harvester_name, object_name)
```

Comment:
Tu by som ocakaval nieco ako
```ruby
min_id = Itms::AccountingDocument.maximum(:itms_identifier).to_i
Itms::AccountingDocumentsHarvester.run(min_id: min_id)

#...

min_id = Itms::SubmittedZop.maximum(:itms_identifier).to_i
Itms::SubmittedZopHarvester.run(min_id: min_id)
```

namiesto tych `to_s` a `constantize` konverzii.

Pripadne ak je ta detekcia `min_id` defaultne chovanie, tak by to tie harvestre mohli detekovat aj same a ja len zavolam `Itms::SubmittedZopHarvester.run`